src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesEdit.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+import { Card, Checkbox, Col, Layout, Row, Select, TextArea, TextField } from '@folio/stripes/components';
+import { required } from '../../../util/validators';
+
+export default class ExternalDataSourcesEdit extends React.Component {
+  static propTypes = {
+    actionButtons: PropTypes.func,
+    input: PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      value: PropTypes.shape({
+        id: PropTypes.string,
+      }).isRequired,
+    }).isRequired,
+    meta: PropTypes.shape({
+      invalid: PropTypes.bool,
+      pristine: PropTypes.bool,
+      submitting: PropTypes.bool,
+    })
+  }
+
+  render() {
+    const { input: { value, name } } = this.props;
+    const kbadapterTypes = [{ value: 'org.olf.kb.adapters.EbscoKBAdapter', label: 'org.olf.kb.adapters.EbscoKBAdapter' }];
+    const recordTypes = [{ value: '1', label: 'Package' }];
+    return (
+      <Card
+        data-test-external-data-source-edit
+        headerStart={(
+          <strong>
+            {value.id ?
+              <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.editExternalKb" />
+              :
+              <FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.newExternalKb" />}
+          </strong>
+        )}
+        headerEnd={this.props.actionButtons}
```

Comment:
If you're destructuring props, destructure these as well. (I now see I made this error in mine as well.)